src/FlowTypes/Types.js

```diff
@@ -0,0 +1,21 @@
+export type Id = mixed;
```

Comment:
Please rename this to index.js and everywhere else, where the types need to be imported, use `import * from '/FlowTypes'` or something like that. The same goes for the Constants folder as well as the Utils folder